docs/tutorial/installation.rst

```diff
@@ -7,6 +7,12 @@ Installation
 This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and ``sunpy``.
 For further information and alternative methods for installing ``sunpy`` beyond the recommended approach outlined below, refer to :ref:`sunpy-topic-guide-installing`.
 
+.. note::
+   We encourage all new and existing users to contribute code to this community project. 
+   Please consider following the :ref:`developer installation instructions <_newcomers>`.
+   It is slightly more difficult and will make it easier to contribute code in the future.
+to do this later!
+
```

Comment:
```suggestion
.. note::

   We encourage all new and existing users to contribute code to this community project. 
   Please consider following the :ref:`developer installation instructions <newcomers>`.
   It is slightly more difficult but will make it easier to contribute code in the future.

```

This link will never work as the newcomers reference only exists in the latest version and never the released version of the documentation.

I also do not see the point of this.